enhancements/plain-k8s-bundles.md

```diff
@@ -0,0 +1,220 @@
+
+Alice has a set of kubernetes manifests describing her operator already: a single CRD, a Service, a Deployment, and some RBAC rules (a fairly straightforward operator deployment). As a user she is able to take these manifests and run an opm command to build a valid bundle. The bundle will include all the necessary metadata for OLM to manage the operator and surface it in a catalog.
+
+OLM is then able to take this bundle and install it in the cluster from a subscription to a catalog source containing the bundle. OLM installs the operator by constructing a CSV representation of the kubernetes objects and then generating an InstallPlan.
+
+The end result is a valid OLM-managed operator deployment running in the cluster, and Alice having a good user experience packaging her existing operator.
+
+#### Story 2
+
+As a pipeline maintainer, Bob wants to package an operator, described as a set of kubernetes manifests, and add it to an index image. The existing index image he is using contains a variety of bundles, both of the `registry+v1` and `k8s+v1` type. Bob expects that by running opm commands he will be able to produce a new index image containing his operator. The catalog source running this image can successfully serve the operator, and the packageserver can surface information on this operator both in-cluster and via the console.
+
+The end result is Bob is able to more easily update and incorporate OLM into his existing pipeline.  
+
+### Risks and Mitigations
+
+One risk is around backporting indexes with mixed mediatypes to older clusters. This needs to be supported to ensure upgrades and downgrades of OpenShift behave as intended. This is outlined further in the Upgrade/Downgrade section below.
+
+Another risk is around operatorhub.io - it will need to be adjusted to support the `k8s+v1` bundles. This work is not scoped as part of this enhancement and is done by a seperate team - how much effort is involved there is unclear.
+
+Security impacts are minimal because the number of kubernetes objects that OLM supports is still the same - just the delivery mechanism (the bundle format) is changing.
+
+## Design Details
+
+### Test Plan
+
+There will be e2e tests alongside unit tests. The e2e test will verify that the functionality is delivered as expected - bundles consisting just of kubernetes manifests are installed and managed on cluster by OLM. Potentially existing tests can be reused but with the newer bundle format.
+
+Outside of core OLM there will also be tests in operator registry to ensure the new bundle format is consistently generated as we expect. Its also critical to test how indexes composed of k8s+v1 bundles will behave.
+
+Additional tests will ideally include the console as well to ensure all the components are supporting the new functionality as expected.
```

Comment:
are you suggesting that the console should be able to support dispalying all manifests? Is that discussed with the console team?

Reply:
No, console uses CSVs and PackageManifests -- this enhancement changes neither API